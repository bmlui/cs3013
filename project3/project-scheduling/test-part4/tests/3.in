0,1
2,5
3,11
4,4