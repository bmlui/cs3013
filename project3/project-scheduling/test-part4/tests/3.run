./scheduler 1 RR tests/3.in 2
