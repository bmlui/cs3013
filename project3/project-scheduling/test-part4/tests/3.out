Execution trace with RR:
t=0: [Job 0] arrived at [0], ran for: [1]
t=2: [Job 1] arrived at [2], ran for: [2]
t=4: [Job 2] arrived at [3], ran for: [2]
t=6: [Job 3] arrived at [4], ran for: [2]
t=8: [Job 1] arrived at [2], ran for: [2]
t=10: [Job 2] arrived at [3], ran for: [2]
t=12: [Job 3] arrived at [4], ran for: [2]
t=14: [Job 1] arrived at [2], ran for: [1]
t=15: [Job 2] arrived at [3], ran for: [2]
t=17: [Job 2] arrived at [3], ran for: [2]
t=19: [Job 2] arrived at [3], ran for: [2]
t=21: [Job 2] arrived at [3], ran for: [1]
End of execution with RR.
Begin analyzing RR:
Job 0 -- Response time: 0  Turnaround: 1  Wait: 0
Job 1 -- Response time: 0  Turnaround: 13  Wait: 8
Job 2 -- Response time: 1  Turnaround: 19  Wait: 8
Job 3 -- Response time: 2  Turnaround: 10  Wait: 6
Average -- Response: 0.75  Turnaround 10.75  Wait 5.50
End analyzing RR.
